mk/spksrc.service.installer

```diff
@@ -203,31 +208,33 @@ postinst ()
             synoshare --add "${SHARE_NAME}" "${SHARE_DESC}" "${SHARE_PATH}" "" "rw" "" 1 0 >> ${INST_LOG}
         fi
 
-        # Add user/group permissions depending if GROUP is set in UI
-        if [ -n "$GROUP" ]; then
-            synoshare --setuser "${SHARE_NAME}" RW + "${EFF_USER}",@"${GROUP}" >> ${INST_LOG} 2>&1
-        else
+        # Add user permission if no GROUP is set in UI
```

Comment:
If GROUP is set then `set_syno_permmissions` will add the permissions for group level, removed this to avoid doing things double :)
This command also applies slightlight different set of permissions (no `C` at end to allow for changing of permissions by GROUP), so 2 sets of permissions would be applied.

Reply:
That's also in the commit message ;)